setup.py

```diff
@@ -5,6 +5,7 @@
 """
 import os
 from abc import abstractmethod
+from distutils.command.clean import clean  # pylint: disable=E0401,E0611
```

Comment:
Could you also add why this is being ignored (copy/paste from kytos/setup.py)? Future Us will be thankful :)